src/main/java/com/bootcamp/demo/business/AccountService.java

```diff
@@ -30,45 +29,60 @@ public void setEncryptor(Encryptor encryptor) {
         this.encryptor = encryptor;
     }
 
-    public boolean accountValidation(Account account){
+    public String accountValidation(Account account) throws ServiceException {
+
+        StringBuilder stringBuilder = new StringBuilder();
         if(account.getName().isEmpty() || account.getPassword().isEmpty() || account.getEmail().isEmpty() || account.getUsername().isEmpty() || account.getAddress().isEmpty() || account.getPhoneNumber().isEmpty() || account.getDateOfBirth().isEmpty())
-            return false;
+            stringBuilder.append("Empty fields");
 
         String expression = "^([A-Z][a-z]*((\\s)))+[A-Z][a-z]*$";
         if(!account.getName().matches(expression))
-            return false;
+            stringBuilder.append("Wrong name format: must contain only letters and must be \"FirstName LastName\"  \n");
 
         expression = "[a-zA-Z0-9]*$";
         if(!account.getUsername().matches(expression))
-            return false;
+            stringBuilder.append("Wrong username format: must contain only digits and letter\n");
 
         if(account.getPhoneNumber().length() != 10)
-            return false;
-
-        for(char c: account.getPhoneNumber().toCharArray()){
-            if(!Character.isDigit(c))
-                return false;
+            stringBuilder.append("Wrong phoneNumber format: must have 10 digits\n");
+        else {
+            for (char c : account.getPhoneNumber().toCharArray()) {
```

Comment:
This might be a good candidate to experiment a bit with Java 8 new features `.stream().anyMatch(...)` in case u want give it a shot. I think code readability will improve a lot